rules/Symfony43/Rector/StmtsAwareInterface/TwigBundleFilesystemLoaderToTwigRector.php

```diff
@@ -107,6 +108,10 @@ private function resolveFileSystemLoaderNew(StmtsAwareInterface $stmtsAware): ?N
                 continue;
             }
 
+            if ($assign->expr->class instanceof Expr) {
+                continue;
+            }
```

Comment:
@TomasVotruba the `new $class` got true on `isObjectType()` so this check is needed here, the `isObjectType()` method itself may need improvement to detect it automatically.

Reply:
@TomasVotruba I created PR to allow cover on `NewTypeResolver` on PR:

- https://github.com/rectorphp/rector-src/pull/4936